code/modules/clothing/head/helmet.dm

```diff
@@ -4,7 +4,7 @@
 	icon_state = "helmet"
 	flags = HEADBANGPROTECT
 	item_state = "helmet"
-	armor = list(melee = 40, bullet = 30, laser = 30,energy = 10, bomb = 25, bio = 0, rad = 0)
+	armor = list(melee = 30, bullet = 30, laser = 30,energy = 10, bomb = 25, bio = 0, rad = 0)

```

Comment:
MrStonedOne> you'll do it wrong
MrStonedOne> nerf helmets to 35 and make riot be 45